subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/AndroidTestProject.groovy

```diff
@@ -56,14 +53,10 @@ class AndroidTestProject implements TestProject {
 
     @Override
     void configure(CrossVersionPerformanceTestRunner runner) {
-        runner.gradleOpts = ["-Xms$memory", "-Xmx$memory"]
     }
 
     @Override
     void configure(GradleBuildExperimentSpec.GradleBuilder builder) {
-        builder.invocation {
-            gradleOpts("-Xms$memory", "-Xmx$memory")
-        }
     }
```

Comment:
Do we still need these methods?

Reply:
The AGP test projects add a Gradle property to enable the IDE mode via this method: `-Pandroid.injected.invoked.from.ide=true`. Maybe we should add this property to the `gradle.properties` file of those projects as well. WDYT?